shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLDALStatementSQLVisitor.java

```diff
@@ -238,7 +243,57 @@ public ASTNode visitShowWarnings(final ShowWarningsContext ctx) {
     
     @Override
     public ASTNode visitResetStatement(final ResetStatementContext ctx) {
-        return new MySQLResetStatement();
+        MySQLStatementParser.ResetPersistContext persistContext = ctx.resetPersist();
+        if (persistContext != null) {
```

Comment:
@flycash Please put null on the left side of the expression.

Reply:
done